asynction/server.py

```diff
@@ -273,7 +273,12 @@ def emit(self, event: str, *args, **kwargs) -> None:
                     f"Event {event} is not registered under the {namespace} namespace"
                 )
 
-            validate_payload(args, message.payload)
+            # tuples are expanded to multiple arguments, everything else is sent
+            # as a single argument
+            payload_args = (
+                args and args[0] if isinstance(args[0], tuple) else (args[0],)
+            )
+            validate_payload(payload_args, message.payload)
```

Comment:
Essentially, fixing the same problem but on the `emit` side (rather than the `on` side).

Reply:
Same logic is implemented in the `python-socketio` client: https://github.com/miguelgrinberg/python-socketio/blob/main/src/socketio/client.py#L402